src/libstd/net/udp.rs

```diff
@@ -180,6 +180,33 @@ impl UdpSocket {
         }
     }
 
+    /// Returns the socket address of the remote peer this socket was connected to.
+    ///
+    /// If the socket isn't connected, it will return an `ErrorKind::NotConnected` error.
```

Comment:
Move this line above line 198 (the second example), similar to the docs of `set_read_timeout`. Also link to the `ErrorKind::NotConnected` variant at `../../std/io/enum.ErrorKind.html#variant.NotConnected`.

Reply:
Fixed in 54bf8e0